utils/nctl/sh/scenarios-upgrades/upgrade_scenario_09.sh

```diff
@@ -163,46 +163,51 @@ function _step_05()
     done
 }
 
-# Step 06: Await 1 era.
+# Step 06: Await 1 era and then start node 6 with trusted hash within the allowed range
 function _step_06()
 {
-    log_step_upgrades 6 "awaiting next era"
-    nctl-await-n-eras offset='1' sleep_interval='5.0' timeout='180' node_id='2'
+    POST_UPGRADE_BLOCK_HASH=${1}
+    log_step_upgrades 6 "awaiting 1 era to start node 6"
+    nctl-await-n-eras offset='1' sleep_interval='5.0' timeout='300' node_id='2'
+    _stage_node_with_trusted_hash "6" "$STAGE_ID" "$ACTIVATION_POINT" "$POST_UPGRADE_BLOCK_HASH" 6
 }
 
-# Step 07: Stage node 6 with post-upgrade trusted hash and node 7 with pre-upgrade trusted hash
+# Step 07: Await 5 eras and then start node 7 with a trusted hash that is too old
```

Comment:
Similar to my other comment, would a node started with the post-upgrade hash that was out of the range fail here? If so we should cover that too.